frontend/src/features/public-form/PublicFormProvider.tsx

```diff
@@ -458,6 +470,74 @@ export const PublicFormProvider = ({
           // TODO (#5826): Toggle to use fetch for submissions instead of axios. If enabled, this is used for testing and to use fetch instead of axios by default if testing shows fetch is more  stable. Remove once network error is resolved
           if (useFetchForSubmissions) {
             return submitStorageFormWithFetch()
+          } else {
```

Comment:
@sebastianwzq we might have selected the wrong code during the rebase just now 🙈